README.md

```diff
@@ -100,6 +100,50 @@ client.set_pool_info("autoscale",
     )
 ```
 
+**Configuration**
+An AzureClient object can be instantiated and initialized with pool, mounted containers and container registries using a configuration file. This is especially useful if the same pool will be used for running multiple batch jobs and experiments. Use the following example to create a configuration file:

```

Comment:
provide a little more information on which components of the config are optional with your new changes and what will happen if certain keys are included (like pulling in the pool name from the config when creating a pool)